src/ReactNativeZoomableView.tsx

```diff
@@ -1079,16 +1075,15 @@ class ReactNativeZoomableView extends Component<
   }
 
   /**
-   * Zooms to a specific location in our view
+   * Zooms to a specific level and/or location in our view
```

Comment:
Not really and/or since zoom value is not optional

Reply:
Location is optional

Reply:
Oh I see what you mean